substanced/folder/generate/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
I'm not talking about putting them in `sdidev`:  I'm talking about putting them in a subdirectory in *this* repo (separate from the actual Python code and assets).  Since we don't mean to be serving them up, they shouldn't be mixed with files we do mean to serve.